CHANGELOG.md

```diff
@@ -1,5 +1,50 @@
 # Change Log
 
+## [vX.X.X](https://github.com/eclipse-iceoryx/iceoryx/tree/v1.0.0) (unreleased)
+
+[Full Changelog](https://github.com/eclipse-iceoryx/iceoryx/compare/v1.0.1...master)
```

Comment:
Really nice to have this on master. What do you think about a `API Breaking Changes` or `Porting Guide` section?